remoteworkitem/trackerquery.go

```diff
@@ -17,7 +17,8 @@ type TrackerQuery struct {
 	// TrackerID is a foreign key for a tracker
 	TrackerID uuid.UUID `gorm:"ForeignKey:Tracker"`
 	// SpaceID is a foreign key for a space
-	SpaceID uuid.UUID `gorm:"ForeignKey:Space"`
+	SpaceID        uuid.UUID `gorm:"ForeignKey:Space"`
+	WorkItemTypeID uuid.UUID `gorm:"ForeignKey:WorkItemType"`
```

Comment:
The foreign key is most likely not `WorkItemType`, is it?

Reply:
please checkout this 72bc5c5

Reply:
What are you doing? Experimental code change?

Reply:
No. The struct tag `gorm:ForeignKey:WorkItemType` was required. After introducing that struct tag, I was able to remove `as work_item_type_id` from the SQL query.